Packs/ServiceNow/ReleaseNotes/2_6_0.md

```diff
@@ -0,0 +1,22 @@
+#### Layouts
```

Comment:
The pack metadata version is 2_5_8, please change accordingly this file.

Reply:
@ilaner Done

Reply:
Conflict with 2.5.8 and got an error that 2.5.9 exists. So switching the metadata and file name back to 2.6.0